setup.cfg

```diff
@@ -72,6 +72,7 @@ dev =
 # W503: line break before binary operator
 # W504: line break after binary operator
 ignore=E402, W503, W504, W605
+max-line-length = 91
```

Comment:
I could bring this number down but if I do Flake8 would complain for some files in this PR.

Other possibility I could try is setting the config for black for shorter lines.

Reply:
Well, I'm just curious about where these numbers come from...  Ten we simply need to decide collectively what we think is best for readability. 
Actually, I'm a bit surprised to see how these things fluctuate across years ;-)

Reply:
For the record there is a quite bit of variability indeed.

Did set things at 90 for now (because 91 is just a weird number).

[flake8_line_length.txt](https://github.com/nilearn/nilearn/files/10661156/flake8_line_length.txt)

Reply:
I think the flake8 default is 79 and that's what we've been using before our PEP8 CI broke

Reply:
Don't feel strongly about any specific value: will change the value and adapt.